include/mbgl/style/conversion/function.hpp

```diff
@@ -200,7 +200,7 @@ struct Converter<CameraFunction<T>> {
 
 template <class T, class V>
 Result<optional<T>> convertDefaultValue(const V& value) {
-    auto defaultValueValue = objectMember(value, "defaultValue");
+    auto defaultValueValue = objectMember(value, "default");
```

Comment:
@jfirebaugh The reason it didn't trigger failures is that the tests are marked `ignored` on native.  I'll submit a PR to gl-js to un-ignore them.